README.md

```diff
@@ -115,6 +115,14 @@ spring:
     name: demo-app
 ```
 
+### OpenTelemetry Collector
+
+The configuration in the application is identical to the Grafana agent. 
+Whenever this documentation refers to "Grafana Agent", the OpenTelemetry Collector configuration is meant as well.
+
+If you the OpenTelemetry Collector, you can also use the recommended [Spring Boot Dashboard](https://grafana.com/grafana/dashboards/18887), which relies on the prometheus naming conventions (e.g. `_seconds` in the metric names).
+
+- [How to configure the OpenTelemtry Collector](https://grafana.com/docs/opentelemetry/collector/send-otlp-to-grafana-cloud-databases/)
 - [How to configure the Grafana Agent](https://grafana.com/docs/opentelemetry/instrumentation/grafana-agent/)
```

Comment:
I think I would move the Grafana Agent doc link to the `Grafana Agent` section 

For the properties, I would say

> Refer to the Properties section for details about configuration properties

and then include the anchor.

Reply:
The yaml snippet on LN 133 is a bit unclear.  Are we indicating that the Collection is also available at the gRPC localhost:4317 default, which is also the Grafana Agent's default if installed running?

Reply:
removed the otel collector section (see last comment)